en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-remote-h2.md

```diff
@@ -1,429 +1,148 @@
-        5.  Add the `commitOnReturn` property and set it to true for all the datasources, including the custom datasources.
-
-            ``` html/xml
-                            <datasource>
-                                 ...
-                                 <definition type="RDBMS">
-                                     <configuration>
-                                           ...
-                                           <defaultAutoCommit>false</defaultAutoCommit>
-                                           <commitOnReturn>true</commitOnReturn>    
-                                           ...
-                                     </configuration>
-                                 </definition>
-                            </datasource>
-            ```
-
-    -   **Configure the connection pool to rollback pending transactions on connection return**
-
-        1.  Navigate to the `<PRODUCT_HOME>/repository/conf/datasources/master-datasources.xml` file.
-        2.  Disable the `defaultAutoCommit` by defining it as false.
-
-        3.  Add the `rollbackOnReturn` property to the datasources.
-
-            ``` html/xml
-                            <datasource>
-                                 ...
-                                 <definition type="RDBMS">
-                                     <configuration>
-                                           ...
-                                           <defaultAutoCommit>false</defaultAutoCommit> 
-                                           <rollbackOnReturn>true</rollbackOnReturn>
-                                           ...
-                                     </configuration>
-                                 </definition>
-                            </datasource>
-            ```
-
-#### Configuring new  datasources to manage registry or user management data
-
-Follow the steps below to configure new datasources to point to the new  databases you create to manage registry and/or user management data separately.
-
-1.  Add a new datasource with similar configurations as the [`WSO2_CARBON_DB` datasource](#ChangingtoRemoteH2-Changingthedefaultdatabase) above to the &lt; `PRODUCT_HOME>/repository/conf/datasources/master-datasources.xml` file. Change its elements with your custom values. For instructions, see [Setting up datasource configurations.](#ChangingtoRemoteH2-Settingupdatasourceconfigurations)
-2.  If you are setting up a separate database to store registry-related data, update the following configurations in the &lt; `PRODUCT_HOME>/repository/conf/registry.xml` file.
-
-    ``` xml
-            <dbConfig name="wso2registry">
-                <dataSource>jdbc/MY_DATASOURCE_NAME</dataSource>
-            </dbConfig>
+    ``` tab="Example"
+    [database.shared_db]
+    type = "h2"
+    url = "jdbc:h2:tcp://localhost/~/shared_db"
+    username = "regadmin"
+    password = "regadmin"
+    driver = "org.h2.Driver"
+    validationQuery = "SELECT 1"
```

Comment:
For H2, we do not need to specify driver and validation query. It is sufficient to specify type, url, username and password